mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
```

Comment:
In the condition `if (!order)`, it's not clear what exactly is being checked. Is it checking for null, undefined, or any falsy value? It would be better to explicitly check for null or undefined to avoid any confusion. Consider changing it to `if (order === null || order === undefined)`.